init-tools.msbuild

```diff
@@ -4,9 +4,10 @@
     <EnableDefaultItems>false</EnableDefaultItems>
     <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
     <BaseIntermediateOutputPath>$(MSBuildThisFileDirectory)Tools/$(BuildToolsPackageVersion)</BaseIntermediateOutputPath>
+    <BuildToolsPackageVersion Condition="'$(AllowWildCard)'=='true'" >1.0-*</BuildToolsPackageVersion>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.DotNet.BuildTools">
+    <PackageReference Include="$(ComponentPackage)">
```

Comment:
My expectation is that we just add one more PackageReference here and not require any changes to init-tools.cmd/sh.